test/testStatuspage.js

```diff
@@ -350,5 +382,5 @@ describe('Testing statuspage', function testStatuspage() {
     assert.ok(await getTimedPromise(() => exitCode1, 'Process did not exit with code 1'));
     api.stop();
     process.exit = originalExit;
-  });
+  }).timeout(10000);
```

Comment:
Can't you default the timeout globally instead of adding `timeout(10000)` everywhere ?